File: c/area.c
#include <stdio.h>
int main() {
    double vet[3];
    for (int i = 0; i < 3; i++)
        scanf("%lf",&vet[i]);

    double total = (vet[0] * vet[2])/2;
    printf("TRIANGULO: %.3lf\n",total);
    
    total = vet[2] * vet[2] * 3.14159;
    printf("CIRCULO: %.3lf\n",total);
    
    total = ((vet[0] + vet[1])*vet[2])/2;
    printf("TRAPEZIO: %.3lf\n",total);
    
    total = vet[1] * vet[1];
    printf("QUADRADO: %.3lf\n",total);
    
    total = vet[0] * vet[1];
    printf("RETANGULO: %.3lf\n",total);        
    return 0;
}

File: py/area.py
vet = [float(x) for x in input().split()]

total = (vet[0] * vet[2])/2
print('TRIANGULO: {:.3f}'.format(total))

total = vet[2] * vet[2] * 3.14159
print('CIRCULO: {:.3f}'.format(total))

total = ((vet[0] + vet[1])*vet[2])/2
print('TRAPEZIO: {:.3f}'.format(total))

total = vet[1] * vet[1]
print('QUADRADO: {:.3f}'.format(total))

total = vet[0] * vet[1]
print('RETANGULO: {:.3f}'.format(total))
